### PR TITLE
🐣 primitives/table: init table

### PR DESCRIPTION
**Pull request by DonGideon**
table is here!!! 

Screen Shot 2019-07-19 at 15 30 34

### CODE REVIEW

File: packages/primitives/table/src/TableBodyCell.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export const TableBodyCell = component(
+  startWithType<TTableBodyCell>()
+)(({ id, borderColor, backgroundColor, children }) => (
+  <td
+    id={id}
+    style={{
+      margin: 0,
+      padding: 0,
+      backgroundColor,
+      ...(borderColor && { borderTopColor: borderColor, borderTopWidth: '1px', borderTopStyle: 'solid' }),
```
Comment (deepsweet) [inline packages/primitives/table/src/TableBodyCell.tsx]: why it applies only for top border?

Comment (DonGideon) [reply packages/primitives/table/src/TableBodyCell.tsx]: in our table there is only topBorder....

Comment (DonGideon) [reply packages/primitives/table/src/TableBodyCell.tsx]: done............ :|

**Comment by DonGideon** [comment]
@deepsweet should we merge it? 🤔

**Comment by deepsweet** [comment]
`@primitives/table@0.1.0`

**Comment by DonGideon** [comment]
Screen Shot 2019-07-25 at 11 27 42

**Pull request merged**